lib/katello/repository_types/python.rb

```diff
@@ -0,0 +1,15 @@
+require 'pulp_python_client'
+
+Katello::RepositoryTypeManager.register('python') do
+  allow_creation_by_user true
+  pulp3_service_class Katello::Pulp3::Repository::Generic
+  pulp3_api_class Katello::Pulp3::Api::Generic
+  pulp3_plugin 'python'
+  partial_repo_path ''
```

Comment:
You might want to just put a `# TODO: ...` here since we'll need to decide on the partial_repo_path (and if it should be defined here) later.